common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -72,6 +80,46 @@ public final void initOverlays() {
         }
     }
 
+    public final void initOverlayGroups() {
+        // Unregister old instances
+        for (Overlay groupedOverlay : groupedOverlayInstances) {
```

Comment:
How can this happen?

Reply:
We recreate all overlay instances, even if we only added a new one. But this can also happen if we remove an overlay.